### PR TITLE
US 1583733 Add missing language IDs - 28

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for command-line tool examples (Svcutil.exe, ComSvcConfig.exe).
- Used "text" for HTTP response example.
- Used "xml" for SOAP 1.x examples.

Contributes to #2192

### CODE REVIEW

File: docs/framework/wcf/feature-details/http-post-and-http-get-requests-for-aspnet-ajax-endpoints.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Windows Communication Foundation (WCF) allows you to create a service that expos
   
 3. Create a new file named service with a .svc extension in the application. Edit this file by adding the appropriate [\@ServiceHost](../../../../docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md) directive information for the service. Specify that the <xref:System.ServiceModel.Activation.WebScriptServiceHostFactory> is to be used in the [\@ServiceHost](../../../../docs/framework/configure-apps/file-schema/wcf-directive/servicehost.md) directive to automatically configure an ASP.NET AJAX endpoint.  
   
-    ```  
+    ```text
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/http-post-and-http-get-requests-for-aspnet-ajax-endpoints.md]: I'd not add here

```suggestion
    ```
```

File: docs/framework/wcf/feature-details/messaging-protocols.md [hunk 2]
```diff
@@ -411,7 +411,7 @@ The following sequence of steps describes the MTOM-specific encoding process:
 
     6. Add an `href` attribute to the `xop:Include` element with the value cid: uri derived from Content-ID header value generated in step 4b. Remove the enclosing "\<" and ">" characters, URL-escape the remaining string, and add the prefix `cid:`. The following minimum character set is required to be escaped by RFC1738 and RFC2396. Other characters can be escaped.
 
-        ```
+        ```xml
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/messaging-protocols.md]: All of these entries on this page are not really xml. Maybe leave without id?

```suggestion
        ```
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Great changes @v-thpra! Left a couple of suggestions for two articles where I think we might leave the code blocks with no identifier.

**Comment by v-thpra** [comment]
Thanks for the careful review, and great feedback @mairaw! I left "xml" on the larger soap/xml examples because it seemed a close enough fit. However, if it just doesn't seem right to you, then it's no problem to take them off too.

**Comment by mairaw** [comment]
I think I'd still remove those.

**Comment by v-thpra** [comment]
Fixed them. Thanks @mairaw.

**Review by HongGit — APPROVED** [review]

**Review by mairaw — APPROVED** [review]

**Pull request merged**